functions/make_plots.R

```diff
@@ -0,0 +1,304 @@
+make_plots <- function(metab.list, stoftest, pt, zscore_cutoff, xaxis_cutoff, ThisProbScore,
+                       ratios_cutoff, ptcount) {
+  stoftest <- as.character(stoftest)
+  if (ncol(metab.list) > 2) {
+    # metab list are the alarm values, so reduce the data frame to 2 columns and save metab.list4
+    metab.list4 <- metab.list
```

Comment:
In the context of this function 2 and 3 do not exists. So starting with number 4 does not make sense.